qiskit/pulse/commands/__init__.py

```diff
@@ -13,5 +13,4 @@
 from .acquire import Acquire, Discriminator, Kernel
 from .frame_change import FrameChange
 from .persistent_value import PersistentValue
-from .functional_pulse import FunctionalPulse
```

Comment:
`pulse_decorators.function` should be imported and made available in top-level pulse namespace

Reply:
I'll do that in separate PR to include all required modules in `qiskit.pulse.__init__` .